wg-etcd-operator/charter.md

```diff
@@ -0,0 +1,60 @@
+# WG etcd operator
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+The purpose of an etcd-operator is to operate automatically etcd clusters which run in the Kubernetes environment.
```

Comment:
IIUC one of the main goals is to cover only operating etcd, but not the one backing the k8s cluster. I think it's important to make this explicit, so that users won't get confused about it. I'm aware it's mentioned as out of scope, but let's repeat it here. I remember when I first heard about it, that particular etcd was what I thought about.